draft-ietf-pquip-pqc-engineers.md

```diff
@@ -456,7 +438,7 @@ Here, `Combiner(ss1, ss2)`, often referred to as a KEM Combiner is a cryptograph
 
 ## Security property
 
-* IND-CCA2 : IND-CCA2 (INDistinguishability under adaptive Chosen-Ciphertext Attack) is an advanced security notion for encryption schemes. It ensures the confidentiality of the plaintext, resistance against chosen-ciphertext attacks, and prevents the adversary from forging new ciphertexts. An appropriate definition of IND-CCA2 security for KEMs can be found in [CS01] and [BHK09]. Kyber and Classic McEliece provide IND-CCA2 security.
+* IND-CCA2 : IND-CCA2 (INDistinguishability under adaptive Chosen-Ciphertext Attack) is an advanced security notion for encryption schemes. It ensures the confidentiality of the plaintext, resistance against chosen-ciphertext attacks, and prevents the adversary from deciphering accurate plaintexts (given access to the public key). An appropriate definition of IND-CCA2 security for KEMs can be found in [CS01] and [BHK09]. Kyber and Classic McEliece provide IND-CCA2 security.
```

Comment:
what do you mean by "accurate plain texts" ?
what is wrong with the previous text saying "prevents the adversary from forging new ciphertexts" (it is saying IND-CCA2 does not allow an attacker to create valid ciphertexts for arbitrary plaintexts without having access to the encryption key).

Reply:
Lets just say "forging valid ciphertexts" instead of "new". That might solve the confusion.